backend/candidate_comparison_tool/service.py

```
@@ -0,0 +1,29 @@
+from backend.resume_evaluations.schemas import ResumeEvaluationRequest
```

Pylint (3.3.7)

[error] No name 'schemas' in module 'backend.resume_evaluations'

(E0611)